### PR TITLE
Update mysql_db.py

### PR DESCRIPTION
**Pull request by aioue**
Without the `mysql` binary on that host running the module, you will get a fatal error.

`"msg": "Failed to find required executable mysql"`

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @aioue. @Jmainguy please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

**Comment by Jmainguy** [comment]
@aioue Nice catch. I think this would go better after line 60 "extends documentation". In the notes/requirements field. Something like.

```
requirements:
   - mysql
   - mysqldump
notes:
   - Requires the mysqldb package on the remote host. the package provides the
     commands mysql, and mysqldump.
```

What do you think?

**Comment by aioue** [comment]
@Jmainguy updated as suggested.

**Comment by Jmainguy** [comment]
@aioue looks great man, see if you can squash the two commits into one using 

```
git rebase -i HEAD~2
```

Or something like that, if not dont stress over it.

@gregdek 'shipit'

**Comment by robynbergeron** [comment]
Thanks again to @aioue for this PR, and thanks @Jmainguy for reviewing. Marking for inclusion.

File: database/mysql/mysql_db.py [hunk 1]
```diff
@@ -57,6 +57,11 @@
         files (C(.sql)) as well as bzip2 (C(.bz2)), gzip (C(.gz)) and xz (Added in 2.0) compressed files are supported.
     required: false
 author: "Ansible Core Team"
+requirements:
+   - mysql
+   - mysqldump
```
Comment (bcoca) [inline database/mysql/mysql_db.py:62]: normally it is assume that these are python libs, i would recommend stating that they are the command line utilities.

**Comment by bcoca** [comment]
could you also pease squash the commits

**Comment by gregdek** [comment]
Thanks @aioue. @Jmainguy please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

**Comment by Jmainguy** [comment]
This is good to ship once bcoca's concerns are addressed

**Comment by gregdek** [comment]
@aioue A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

**Comment by aioue** [comment]
@gregdek I'm afraid I don't have the git chops to squash the 2 commits into 1 on an edit I've done directly in GitHub's UI.

**Comment by aioue** [comment]
Opened new PR with single commit.

**Comment by Jmainguy** [comment]
@aioue Go ahead and close this, as we are tracking in new PR.

**Pull request closed without merging**